components/contact/Contact.jsx

```diff
@@ -25,7 +25,7 @@ function Contact() {
       </div>
       <div>
         <TextHeading>Contact Us</TextHeading>
-        <ContentBody maxWidth="sm">
+        <ContentBody maxWidth="684px">
```

Comment:
is there a reason to prefer a hard coded value here as opposed to the breakpoint we had before?

Reply:
Hi Jason,  I did this so that the sentence would not overflow on 2 lines.  I went ahead and just removed the maxWidth so that wouldn't occur.